stglib/eofe.py

```diff
@@ -705,14 +740,32 @@ def average_burst(ds):
     return ds
 
 
-def ds_swap_dims(ds):  # swap vert dim to z
+def ds_swap_dims(
+    ds,
+):  # swap vert dim to z or dim specified by vert_dim in config yaml file
     # need to preserve z attrs because swap_dims will remove them
-    attrsbak = ds["z"].attrs
-    for v in ds.data_vars:
-        if "bins" in ds[v].coords:
-            ds[v] = ds[v].swap_dims({"bins": "z"})
 
-    ds["z"].attrs = attrsbak
+    if "vert_dim" in ds.attrs:
+        vdim = ds.attrs["vert_dim"]
```

Comment:
Is this something that is input by the user in a config file, or does it come from the instrument? If the former, it should be in the docs.

Reply:
vert_dim is a new user provided option in the config.yaml file. I will update the docs to include. Right now this option is only available in the eofe.py, but plan to add other instruments in the future. I suggest that be done for other profiling instruments in a separte pull request after this one is merged.